catroid/src/main/java/org/catrobat/catroid/ui/BaseActivity.java

```diff
@@ -174,6 +178,15 @@ private boolean isRecoveredFromCrash() {
 		return PreferenceManager.getDefaultSharedPreferences(this).getBoolean(RECOVERED_FROM_CRASH, false);
 	}
 
+	private Intent createSettingsIntent() {
```

Comment:
since this change in funcitonality is depending solely on the flavor, why not cover this also by moving the SettingsActivity into the flavor folders?